lib/core/src/lib/datatable/data-column/data-column.component.stories.ts

```diff
@@ -240,26 +225,102 @@ export default {
         },
         type: {
             description:
-                'Value type for the column. Possible settings are `text`, `image`, `date`, `fileSize`, `location`, and `json`.',
-            control: { disable: true },
+                'Value type for the column. Possible settings are: `text`, `icon`, `image`, `date`, `fileSize`, `location`, `boolean`, `amount`, `number` and `json`.',
+            control: { type: 'select', disable: false },
+            options: [
+                'text',
+                'icon',
+                'image',
+                'date',
+                'fileSize',
+                'location',
+                'boolean',
+                'amount',
+                'number',
+                'json'
+            ],
             table: {
                 category: 'Component Inputs',
                 type: {
-                    summary: 'string'
+                    summary: 'DataColumnType'
                 },
                 defaultValue: {
                     summary: 'text'
                 }
             },
             defaultValue: 'text'
         },
+        currencyConfig: {
+            description:
+                `The currencyConfig input allows you to customize the formatting and display of currency values within the component.`,
+            control: { type: 'object', disable: true },
```

Comment:
Disable editing currencyConfig - we will see only `-` . The strategy is to enable this only in case of amount column story. Below how we do it:
```
export const amountColumn: Story = template.bind({});
amountColumn.argTypes = {
    copyContent: { control: { disable: true } },
    currencyConfig: { control: { disable: false } }
};
```
For remaining stories we will see just `-`